ui/apps/profile/src/components/pages/profile-info-page.tsx

```diff
@@ -85,7 +85,8 @@ const ProfileInfoPage: React.FC<RootComponentProps> = props => {
 
   const handleCopy = () => {
     /** handle copy */
-    const profileUrl = `${location.origin}${location.pathname}`;
+
+    const profileUrl = new URL(`${location.origin}${location.pathname}`).href;
```

Comment:
check the documentation if you're not sure, the segments should be passed individually to the URL constructor, this is not the way what you've implemented here